Sming/Arch/Host/readme.md

```diff
@@ -0,0 +1,158 @@
+# Sming Host Emulator

```

Comment:
Update the readme with information from https://github.com/SmingHub/Sming/wiki/Host-Emulator. For example the tap0 setup for Linux has changed.